src/cruise_control/scripts/pid_controller.py

```diff
@@ -14,23 +14,23 @@ class PIDController:
 
     def __init__(self):
         # This depends on exp_factor in pidToCarValues()
-        self.speedAdjustmentFactor = 1.15
-        self.currentSpeed = 0.0
-        self.targetSpeed = 5.0 * self.speedAdjustmentFactor
+        self.speed_adjustment_factor = 1.15
+        self.current_speed = 0.0
+        self.target_speed = 7.0 * self.speed_adjustment_factor
```

Comment:
Forgot to change back to 5 m/s, as it stands car just completes given path faster